analyzers/src/SonarAnalyzer.Common/Rules/ClassNotInstantiatableBase.cs

```diff
@@ -31,20 +31,18 @@ public abstract class ClassNotInstantiatableBase : SonarDiagnosticAnalyzer
         protected const string MessageFormat = "This {0} can't be instantiated; make {1} 'public'.";
 
         protected abstract bool IsTypeDeclaration(SyntaxNode node);
+        protected abstract bool IsObjectCreation(SyntaxNode node);
 
-        protected bool IsAnyConstructorCalled
-            <TBaseTypeSyntax, TObjectCreationSyntax>
-            (INamedTypeSymbol namedType, IEnumerable<SyntaxNodeAndSemanticModel<TBaseTypeSyntax>> typeDeclarations)
-            where TBaseTypeSyntax : SyntaxNode
-            where TObjectCreationSyntax : SyntaxNode =>
+        protected bool IsAnyConstructorCalled<TBaseTypeSyntax>(INamedTypeSymbol namedType, IEnumerable<SyntaxNodeAndSemanticModel<TBaseTypeSyntax>> typeDeclarations)
+            where TBaseTypeSyntax : SyntaxNode =>
             typeDeclarations
                 .Select(typeDeclaration => new
                 {
                     typeDeclaration.SemanticModel,
                     DescendantNodes = typeDeclaration.SyntaxNode.DescendantNodes().ToList()
                 })
                 .Any(descendants =>
-                    IsAnyConstructorToCurrentType<TObjectCreationSyntax>(descendants.DescendantNodes, namedType, descendants.SemanticModel) ||
+                    IsAnyConstructorToCurrentType(descendants.DescendantNodes, namedType, descendants.SemanticModel) ||
```

Comment:
Let's fix the `||` formatting, it's modified line and it's the only occurrence in this file